apps/admin-server/src/pages/projects/[project]/duplicate.tsx

```diff
@@ -48,6 +49,27 @@ export default function ProjectDuplicate() {
 
   if (!data) return null;
 
+  async function fetchData( url: string) {
+    let response = await fetch(url) || [];
+    if (response.ok) {
+      let data = await response.json();
+
+      data = Array.isArray(data) ? data.map((item) => {
+        if (item.deletedAt) {
+          return null;
+        }
+        delete item.projectId;
+
+        item.originalId = item.id;
+        delete item.id;
+
+        return item;
+      }) : [];
+
+      return data;
```

Comment:
Dit kun je ook zonder ternary oplossen dmv een guard clause.

```suggestion
      if (!Array.isArray(data)) {
        return data;
      }

      return data.map((item) => {
        if (item.deletedAt) {
          return null;
        }
        delete item.projectId;

        item.originalId = item.id;
        delete item.id;

        return item;
      })
```